plag.py

```diff
@@ -34,7 +34,7 @@ def cosineSimilarity():
 	database1 = fd.read().lower()
 
 	databaseWordList = re.sub("[^\w]", " ",database1).split()	#Replace punctuation by space and split
-	databaseWordList = map(str, databaseWordList)
+	# databaseWordList = map(str, databaseWordList)			#And this also leads to divide by zero error
```

Comment:
map can be iterated only once and we are iterating this object twice hence resulting in divide by zero error